code/release-2/00-metadata/emp500_s2_add_prep_info_sample_names.ipynb

```diff
@@ -0,0 +1,252 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## emp500_s2_add_prep_info_sample_names.ipynb"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Define function"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def add_sample_ids(infile_samples, infile_prep, outfile_prep, drop_sample_name_plus_plate):\n",
+    "    df_samples = pd.read_excel(infile_samples)\n",
+    "    df_samples = df_samples[['#SampleID', 'sample_name_plus_plate']]\n",
+    "    df_prep = pd.read_excel(infile_prep)\n",
+    "    df_merged = pd.merge(df_samples, df_prep, left_on='sample_name_plus_plate', right_on='SampleID', how='inner')\n",
+    "    df_merged.drop('SampleID', axis=1, inplace=True)\n",
+    "    if drop_sample_name_plus_plate == True:\n",
+    "        df_merged.drop('sample_name_plus_plate', axis=1, inplace=True)\n",
+    "    df_merged.to_excel(outfile_prep, index=False)\n",
+    "    print(df_prep.shape[0], df_merged.shape[0])\n",
+    "    #print(df_prep.columns[:5])\n",
+    "    #print(df_merged.columns[:5])\n",
+    "    #return sorted(list(set(df_prep['SampleID']) - set(df_merged['sample_name_plus_plate'])))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### File names"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "input_sampleIDs = '/Users/luke.thompson/emp/500-metadata/input-sample-prep-info/emp500_sample_information_manual.xlsx'\n",

```

Comment:
Same comment here with the path

But also same general question, why are we doing a bunch of work to integrate prep info when we can just download from qiita?